src/cmake/externalpackages.cmake

```diff
@@ -43,11 +52,17 @@ checked_find_package (TIFF REQUIRED
 
 # IlmBase & OpenEXR
 checked_find_package (Imath REQUIRED
-                      VERSION_MIN 3.1
-                      PRINT IMATH_INCLUDES Imath_VERSION)
+    VERSION_MIN 3.1
+    BUILD_LOCAL missing
```

Comment:
Just and observation: BUILD_LOCAL could potentially be an option for Windows builds and be used for `REQUIRED` dependencies. It could simplify the process discussed in [3816](https://github.com/AcademySoftwareFoundation/OpenImageIO/issues/3816#issuecomment-1519172373)  and avoid falling back on Bash, vcpkg etc. Do you see it as a use case?

Reply:
Yes, definitely